CoffeeChat/Controller/SchedulingPlacesViewController.swift

```diff
@@ -274,24 +311,41 @@ extension SchedulingPlacesViewController: UICollectionViewDataSource {
         case .campus(let locations):
             guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: campusReuseIdentifier, for: indexPath) as?
             SchedulingPlaceCollectionViewCell else { return UICollectionViewCell() }
-            cell.configure(with: locations[indexPath.row])
+            cell.configure(with: locations[indexPath.row], isOtherSide: isOtherSide)
             return cell
         case .ctown(let locations):
             guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ctownReuseIdentiifier, for: indexPath) as?
             SchedulingPlaceCollectionViewCell else { return UICollectionViewCell() }
-            cell.configure(with: locations[indexPath.row])
+            cell.configure(with: locations[indexPath.row], isOtherSide: isOtherSide)
             return cell
         }
     }
 
     func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
-        let header = indexPath.row == 0
+        let header = indexPath.section == 0
```

Comment:
can we do `let isCampus = indexPath.section == 0` so that its clear what it means